src/components/ReportActionItemIOUPreview.js

```diff
@@ -33,6 +32,9 @@ const propTypes = {
 
         // Outstanding amount of this transaction
         cachedTotal: PropTypes.string,
+
+        // Is the IOU report settled?
```

Comment:
"settled" is kind of an ambiguous term and I'm not sure what it means. Maybe the comment could clarify it better like "Is there an unpaid IOU?"

Reply:
Yep, have rewritten these.